dotnet/dotnet-cloud-run-hello-world/README.md

```diff
@@ -26,18 +26,21 @@ This sample demonstrates how to use the Cloud Code extension in IntelliJ.
 
 ### IntelliJ Running on Cloud Run
 
-1. Select `Edit Configurations` from the Run/Debug configurations dialog on the
-  top taskbar.
-1. Add a `Cloud Code: Cloud Run: Deploy` configuration.
-1. Select `Cloud Code: Deploy` target from the Run/Debug configurations
-  dialog on the top taskbar.
+1. Enable Cloud Run support on your project.
+In the Console, use the navigation menu to access `APIs and Services` > `Dashboard`.
```

Comment:
would be great to link "Console" to the Pantheon root on all of these. The quickstart links here:
http://console.cloud.google.com/

Reply:
SG, will do.